BUILD.gn

```diff
@@ -372,67 +375,78 @@ action("create_symbols_dist") {
 
 action("create_dist_zips") {
   output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_arch.zip"
+  outputs = [ output ]
 
-  script = "//brave/script/create-dist.py"
+  if (is_win) {
+    # Repack a chrome release archive to the brave format.
```

Comment:
nit: `to the brave format`. maybe just `from 7z to zip`? Currently it looks like something shady is going on in `repack-archive.py`, but it's just an archive format change step and nothing more.

Reply:
Not exactly. We change the directory structure: `Chrome-bin/brave.exe` => `brave.exe`
The idea is the `brave format` refers to this structure, not the file format.

Another thing: if we want to support another OS, we may change more things in the archive.

Reply:
okay, let's keep it as is then.

Reply:
maybe it could be:
```
# Repack a Chrome release archive to the Brave format.
# Besides changing from 7z to zip, the directory structure is a bit different.
```